inc/projector.h

```diff
@@ -20,7 +20,7 @@
 #include <deal.II/dofs/dof_tools.h>
 #include <deal.II/fe/mapping.h>
 #include <deal.II/fe/mapping_q1.h>
-#include <deal.II/lac/constraint_matrix.h>
+#include <deal.II/lac/affine_constraints.h>
```

Comment:
Could you explain this change? 
Is constraint_matrix.h a deprecated version of affine_constraints.h ?

Reply:
Yes, that's exactly it. As of 9.1 it is deprecated, and does nothing but include the new file, so I made the change.

Reply:
Thanks. It makes sense.